src/Form/ColumnsExtractor.php

```diff
@@ -33,6 +38,32 @@ public function getColumnLabel($formElement)
         return $formElement->getConfig()->getOptions()['label'];
     }
 
+    /**
+     * Method can be overwritten in order to make custom name for columns.
+     *
+     * @param \Symfony\Component\Form\Form $formElement
+     *
+     * @return string name for column
+     */
+    public function getColumnName($formElement)
+    {
+        return $formElement->getConfig()->getName();
+    }
+
+    /**
+     * Method can be overwritten in order to make custom style for columns.
+     *
+     * @param \Symfony\Component\Form\Form $formElement
+     *
+     * @return string style for column
+     */
+    public function getColumnStyle($formElement)
```

Comment:
Maybe add type hint?

Reply:
as discussed, we can leave it without type hinting for more flexibility